app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -385,6 +361,7 @@
             return item.primStat &&
               item.primStat.statHash === 3897883278 && // has defence hash
               ((vm.showBlues && item.tier === 'Rare') || item.tier === 'Legendary' || (vm.showExotics && item.tier === 'Exotic')) &&
+              item.primStat.value >= 280 // only 280+ light items
```

Comment:
i added this in also. only look at items >= 280 defense. reasoning is we only can predict items at 335 if they are >= 280.